### ISSUE TITLE
html redirection to outline node

### ISSUE THREAD

**Issue by eroux** [buda-base/lds-pdi#185]
http://purl.bdrc.io/resource/MW12827_946A0E

should redirect to

http://library-dev.bdrc.io/show/bdr:MW12827?part=bdr:MW12827_946A0E

which means a bit of querying

**Comment by MarcAgate** [comment]
What is the exact general rule? Will this always follow that pattern: 

http://purl.bdrc.io/resource/AAAAA_BBBBB

becomes

http://library-dev.bdrc.io/show/bdr:AAAAA?part=bdr:AAAAA_BBBBB

**Comment by eroux** [comment]
no, 

http://purl.bdrc.io/resource/AAA

becomes

http://library-dev.bdrc.io/show/bdr:BBB?part=bdr:AAA

when the following triple exists:

```
bdr:AAA :inRootInstance bdr:BBB .
```

**Comment by MarcAgate** [comment]
Done as of commit feb253e

Note: this is not deployed on purl.bdrc.io + redirection to library.bdrc.io

it is deployed on ldspdi-dev.bdrc.io + redirection to library-dev.bdrc.io (check http://ldspdi-dev.bdrc.io/resource/MW12827_946A0E )

Unless there are some reasons to do otherwise, I will not publish anything in PROD anymore (here is an example of a feature that is only applicable to library-dev and therefore there is no point in deploying it to purl.bdrc.io)

**Comment by eroux** [comment]
Are you sure you pointed to the right commit? Also, I don't understand your last paragraph, can you rephrase it?

**Comment by MarcAgate** [comment]
It was the commit before (the last one just disable local logback I am using locally). The actual commit is the one before: commit ce9a3b3

What I mean is that http://library.bdrc.io/show/bdr:MW12827?part=bdr:MW12827_946A0E does nothing since it's the prod version of library, not the one that Nicolas is working on and that produces the expected redirection outcome: 
http://library-dev.bdrc.io/show/bdr:MW12827?part=bdr:MW12827_946A0E

Since purl.bdrc.io (Prod) works with library.bdrc.io (prod), there is no point in updating purl.bdrc.io unless Nicolas decides to update library.bdrc.io with the latest library-dev code.

**Comment by eroux** [comment]
oh ok I see yes, we'll update both ldspdi and library when the outlines are done
